### PR TITLE
Polish SCIM setup

### PR DESCRIPTION
**Pull request by jwitz**

### CODE REVIEW

File: software/integrate-auth-system.md [hunk 1]
```diff
@@ -422,27 +422,43 @@ See [Add SCIM provisioning to app integrations](https://help.okta.com/en-us/Cont
 
 <TabItem value="azuread">
 
-This setup assumes you have already registered Astronomer as an application on Azure AD. See [Azure AD](#azure-ad).
+This setup assumes you have already registered Astronomer Software as an application on Azure AD. See [Azure AD](#azure-ad).
 
-1. Go to **Manage** > **Provisioning**.
+1. If user provisioning isn't available on your existing Astronomer Software application, create a new application using the **Azure Databricks SCIM Provisioning Connector**. See [Configure SCIM provisioning using Microsoft Azure Active Directory](https://learn.microsoft.com/en-us/azure/databricks/administration-guide/users-groups/scim/aad#step-2-configure-the-enterprise-application).
+   
+2. In the menu for your configured application, go to **Manage** > **Provisioning**.
```
Comment (jwitz) [inline software/integrate-auth-system.md:429]:
```suggestion
2. In the menu for your Azure application, go to **Manage** > **Provisioning**.
```

File: software/integrate-auth-system.md [hunk 2]
```diff
@@ -422,27 +422,43 @@ See [Add SCIM provisioning to app integrations](https://help.okta.com/en-us/Cont
 
 <TabItem value="azuread">
 
-This setup assumes you have already registered Astronomer as an application on Azure AD. See [Azure AD](#azure-ad).
+This setup assumes you have already registered Astronomer Software as an application on Azure AD. See [Azure AD](#azure-ad).
```
Comment (kwcanuck) [inline software/integrate-auth-system.md:425]: When I checked this link it took me to the Azure AD section. So far, so good. However, I noticed that there were two headings with no between them:

image

The following are the recommendations from our style guides:

image

image

File: software/integrate-auth-system.md [hunk 3]
```diff
@@ -422,27 +422,43 @@ See [Add SCIM provisioning to app integrations](https://help.okta.com/en-us/Cont
...
 
-1. Go to **Manage** > **Provisioning**.
+1. If user provisioning isn't available on your existing Astronomer Software application, create a new application using the **Azure Databricks SCIM Provisioning Connector**. See [Configure SCIM provisioning using Microsoft Azure Active Directory](https://learn.microsoft.com/en-us/azure/databricks/administration-guide/users-groups/scim/aad#step-2-configure-the-enterprise-application).
```
Comment (kwcanuck) [inline software/integrate-auth-system.md:427]:
```suggestion
1. If user provisioning isn't available on your existing Astronomer Software application, create a new application using the **Azure Databricks SCIM Provisioning Connector**. See [Configure SCIM provisioning using Microsoft Azure Active Directory](https://learn.microsoft.com/en-us/azure/databricks/administration-guide/users-groups/scim/aad#step-2-configure-the-enterprise-application).
```

Why is Azure Databricks SCIM Provisioning Connector bold? Is this a UI element? If it isn't, there is no need for bold and the capitalization should be updated to: Azure Databricks SCIM provisioning connector.

File: software/integrate-auth-system.md [hunk 4]
```diff
@@ -422,27 +422,43 @@ See [Add SCIM provisioning to app integrations](https://help.okta.com/en-us/Cont
...
+   
+2. In your Astronomer Software application on Azure AD, go to **Manage** > **Provisioning**.
+
+3. In the **Provisioning Mode** menu, select **Automatic**.
```
Comment (kwcanuck) [inline software/integrate-auth-system.md:431]:
```suggestion
3. Click **Provisioning Mode** > **Automatic**.
```

File: software/integrate-auth-system.md [hunk 5]
```diff
@@ -422,27 +422,43 @@ See [Add SCIM provisioning to app integrations](https://help.okta.com/en-us/Cont
...
 
-2. In the **Provisioning Mode** menu, select **Automatic**.
+4. In **Tenant URL**, enter the URL for Astronomer's SCIM endpoint. This should be: `https://BASEDOMAIN.astronomer.io/v1/scim/microsoft`.
```
Comment (kwcanuck) [inline software/integrate-auth-system.md:433]:
```suggestion
4. In the **Tenant URL** field, enter `https://BASEDOMAIN.astronomer.io/v1/scim/microsoft`. This is the Astronomer SCIM endpoint URL.
```

File: software/integrate-auth-system.md [hunk 6]
```diff
@@ -422,27 +422,43 @@ See [Add SCIM provisioning to app integrations](https://help.okta.com/en-us/Cont
...
+7. In the **Provisioning** menu for your Azure application, paste your token into the **Secret Token** field. If this field is left blank, Azure AD includes an OAuth bearer token issued from Azure AD with each request.
 
-5. Click **Test connection** to confirm your connection to the SCIM endpoint.
+8. Click **Test connection** to confirm your connection to the SCIM endpoint.
 
-6. Create mappings for your Astronomer users and roles. See [Tutorial - Customize user provisioning attribute-mappings for SaaS applications in Azure Active Directory](https://learn.microsoft.com/en-us/azure/active-directory/app-provisioning/customize-application-attributes).
+9.  Create mappings for your Astronomer users and roles. See [Tutorial - Customize user provisioning attribute-mappings for SaaS applications in Azure Active Directory](https://learn.microsoft.com/en-us/azure/active-directory/app-provisioning/customize-application-attributes).
 
-7. Go to **Manage** > **Provisioning** > **Settings**.
+10. Go to **Manage** > **Provisioning** > **Settings**.
```
Comment (kwcanuck) [inline software/integrate-auth-system.md:455]:
```suggestion
10. Click **Manage** > **Provisioning** > **Settings**.
```

File: software/integrate-auth-system.md [hunk 7]
```diff
@@ -422,27 +422,43 @@ See [Add SCIM provisioning to app integrations](https://help.okta.com/en-us/Cont
...
 
-8. In the **Scope** setting, select **Sync only assigned users and groups**.
+11. In the **Scope** setting, select **Sync only assigned users and groups**.
```
Comment (kwcanuck) [inline software/integrate-auth-system.md:457]:
```suggestion
11. In the **Scope** setting list, select **Sync only assigned users and groups**.
```

File: software/integrate-auth-system.md [hunk 8]
```diff
@@ -422,27 +422,43 @@ See [Add SCIM provisioning to app integrations](https://help.okta.com/en-us/Cont
...
    
-9. Turn the **Provisioning status** toggle to **On**.
+12. Turn the **Provisioning status** toggle to **On**.
```
Comment (kwcanuck) [inline software/integrate-auth-system.md:459]:
```suggestion
12. Click the *Provisioning status** toggle to turn provisioning status on.
```

image

**Review by kwcanuck — APPROVED** [review]

**Review by ajayy004 — APPROVED** [review]

File: software/integrate-auth-system.md (hunk 3 above)
Comment (jwitz) [reply software/integrate-auth-system.md:427]: It is a UI element according to the Azure docs

Screen Shot 2022-12-09 at 3 07 26 PM

**Pull request merged**